keyboards/peej/lumberjack/rules.mk

```diff
@@ -16,5 +16,6 @@ NKRO_ENABLE = no            # Enable N-Key Rollover
 BACKLIGHT_ENABLE = no       # Enable keyboard backlight functionality
 RGBLIGHT_ENABLE = no        # Enable keyboard RGB underglow
 AUDIO_ENABLE = no           # Audio output
+DEFERRED_EXEC_ENABLE = yes
```

Comment:
I would move the defer code into the `default` keymap instead.
1. Users maybe don't want the debugging tool on their final code
1. Adding features to the "base" code may make some user's keymap bigger enough to not fit in the constrained MCU